src/Scripts/Ecs/Systems/DisplaySystem.gd

```diff
@@ -4,11 +4,10 @@ var _tiles_tilemap:TileMap
 var _entities_tilemap:TileMap
 
 var tilemaps_by_name = {}
-var _fov_cache = {} # [x, y] => true/false
-var _fog_of_war_seen_tiles = {} # [x, y] instances for O(1) search
```

Comment:
Why did you remove these extremely helpful comments? I has a sad. (I added them because it's not obvious why we're not using a simple list/array here)

Reply:
They got lost between iterations and copypasting. The first one is outdated, with the new FOV system. I'll bring back the fog of war comment, but update it to reference Vector2 which we now use as keys.